h/static/scripts/directive/test/util.js

```diff
@@ -1,13 +1,40 @@
 'use strict';
 
-// converts a camelCase name into hyphenated ('camel-case') form,
-// as Angular does when mapping directive names to tag names in HTML
+/**
+ * Converts a camelCase name into hyphenated ('camel-case') form.
+ *
+ * This matches how Angular maps directive names to HTML tag names.
+ */
 function hyphenate(name) {
   var uppercasePattern = /([A-Z])/g;
   return name.replace(uppercasePattern, '-$1').toLowerCase();
 }
 
 /**
+ * Helper for retrieving an Angular module in a test.
+ *
+ * Given the 'inject' function from the 'angular-mocks' module,
+ * retrieves an instance of the specified Angular module.
+ */
+function ngModule(inject, name) {
```

Comment:
Yes. I was asking for functions like that to be defined in `util` and used in the tests instead, because it's more readable. This pr has actually replaced `getRootScope()` etc in tests with `util.ngModule(inject, '$rootScope').$new()` which I think is far less readable.